src/core/external/reveal/reveal-element.ts

```diff
@@ -139,6 +169,9 @@ class RevealElement extends SkyflowElement {
     });
   }
 
+  /**
+  * Clears the altText, causing the element to display the token or actual value of the element. If the element has no token, the element becomes empty.
```

Comment:
Clears the alt text and displays the token or data of the element. If the element doesn't have a token, the element becomes empty.